### PR TITLE
Initial Codebase Refactor

### PR DESCRIPTION
**Pull request by BettridgeKameron**
Hello @agentz101 @2012-Pai-Raymond and @Leung-Alvin 

This is an example of a pull request, which is what we can use to review each others added features, before we merge your branch to main! You can suggest changes, review the code, and other aspects. Once a consensus is made that the code is good and working, we can then do a merge!

Remember that you should never be directly modifying the main branch! This is where many issues can arise. Whenever you are working on a feature, make a branch for it! When its done, you can open a PR (Pull Request), and invite everyone else as a reviewer. Do not automatically merge, unless you checked in with everyone please! 

By doing this we will be able to work together and independently, without having to wait for each other to do X thing or Y thing (well can still happen, but once we start splitting tasks down for eachother, we should be fine).

In the future we may have automated tests (CICD/Github Actions) that will automatically check and verify that aspects of the code is functional!

To test, look at the changes, make comments if you wish, and then approve or suggest modifications if you think something needs to be changed.

### CODE REVIEW

**Review by BettridgeKameron — COMMENTED** [review]
To write a review like I am, go to the "Files changed" tab under the Pull request, and then you will see the "Review changes" button where you can comment (what I am doing), Approve (I cant do this as I made this PR), or Request changes to be made.

If a change is requested, the author of the PR can make that change and repush the changes.

_**A new PR does not need to be made. Changes will automatically be added when pushed to the branch**_

Please Approve or Request a change, after you view each file.

File: README.md [hunk 1]
```diff
@@ -1,12 +1,60 @@
...
-Libraries needed:
-nltk
-os
-pandas
-pickle
-random
-math
-shutil
-tkinter
+UNR CS 425/426 Project
```
Comment (BettridgeKameron) [inline README.md:2]: This is how you can add a comment to a specific part of the code.

I will add a more in depth description later of the CSFA.

Comment (BettridgeKameron) [reply README.md:2]: After reading, you can then resolve the conversation when either the comment is read, or you reach a consensus on whatever change was suggested or whatever

**Review by Leung-Alvin — APPROVED** [review]
LGTM. Haven't tested the code out, but things look formatted great.

**Review by agentz101 — APPROVED** [review]
LGTM.  For the code I have done and understand it looks great.

**Comment by 2012-Pai-Raymond** [comment]
Yeah, it seems to look go so far.

**Comment by BettridgeKameron** [comment]
@2012-Pai-Raymond > Yeah, it seems to look go so far.

Make sure to approve the changes of course in the future. Ill go ahead and merge now though assuming you forgot.

**Pull request merged**